pkg/workloadapi/workload.proto

```diff
@@ -258,6 +263,8 @@ message Workload {
 
   NetworkMode network_mode = 25;
 
+  repeated Extension extensions = 26;
```

Comment:
Should this be `map<string, google.protobuf.Any>`?

Reply:
I think a list is more flexible and consistent with Envoy. Probably both are fine but a list is the most flexible and simplest so seems like a better approach